packages/perseus/src/styles/widgets/expression.less

```diff
@@ -69,80 +69,3 @@
         }
     }
 }
-
-// Old expression widget
-.perseus-widget-expression-old {
-    &,
-    > .output,
-    > .output > .tex,
-    > .output > .placeholder {
-        display: block;
-    }
-}
-
-.perseus-widget-expression-old input,
-#answer_area .perseus-widget-expression-old input {
-    direction: ltr;
-    border: 1px solid #a4a4a4;
-    border-radius: 5px;
-    box-sizing: border-box;
-    font-size: 14px;
-    margin-bottom: 5px;
-    max-width: 240px;
-    padding: 6px;
-    width: 100%;
-}
-
-.perseus-widget-expression-old > .output {
-    background: #f2f2f2;
-    border-radius: 5px;
-    padding: 10px;
-    margin: 10px 0;
-}
-
-#answer_area .perseus-widget-expression-old > .output {
-    background: #f7f7f7;
-}
-
-.perseus-widget-expression-old > .output > .tex {
-    overflow-x: auto;
-    padding: 5px;
-}
-
-.perseus-widget-expression-old > .output > .placeholder {
-    position: relative;
-    height: 40px;
-    overflow-y: hidden;
-}
-
-.perseus-widget-expression-old > .output > .placeholder > .error {
-    border-radius: 10px;
-    background: #f7f7f7;
-    border: 1px solid #ddd;
-    box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.5);
-    color: #000;
-    display: none;
-    font-weight: normal;
-    min-height: 22px;
-    width: 168px;
-    position: absolute;
-    top: 0px;
-    left: 40px;
-    margin: auto;
-}
-
-.perseus-widget-expression-old > .output > .placeholder > .error > .buddy {
-    background-image: url(/images/perseus/error-buddy.png);
-    background-size: 100%;
-    height: 36px;
-    width: 40px;
-    position: absolute;
-    top: -3px;
-    left: -42px;
-}
-
-.perseus-widget-expression-old > .output > .placeholder > .error > .message {
-    font: 12px @baseFontFamily;
```

Comment:
While investigating `@baseFontFamily`, I noticed this seemingly unused class. Couldn't find it in Perseus, Webapp, or Mobile.